dev-python/spyder/spyder-4.0.1.ebuild

```diff
@@ -1,67 +1,64 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
 
-PYTHON_COMPAT=( python3_6 )
+PYTHON_COMPAT=( python3_{6,7} )
 
-inherit eutils distutils-r1 xdg-utils
-MY_P=${P/_beta/b}
+inherit eutils distutils-r1 xdg-utils desktop
 
 DESCRIPTION="The Scientific PYthon Development EnviRonment"
```

Comment:
I checked how upstream writes their slogan, and looks like P**Y**thon ... Envi**R**onment are typoes.